addons/website/static/src/js/menu/content.js

```diff
@@ -966,7 +970,13 @@ var ContentMenu = websiteNavbarData.WebsiteNavbarActionWidget.extend({
 
         // If not, write on the server page and reload the current location
         var vals = {};
-        vals[params.name] = option.value;
+        if (params.name === "header_inline_color" || params.name === "header_color") {
+            let toggle_option =  params.name === 'header_inline_color' ? 'header_color' : 'header_inline_color';
+            vals[params.name] = option.value;
+            vals[toggle_option] = '';
+        } else {
+            vals[params.name] = option.value;
+        }
```

Comment:
Ok, you did that here... no, do that in the option directly, by calling
```javascript
        this.trigger_up('action_demand', {
            actionName: 'toggle_page_option',
```
twice. In a future update, I may have an usecase which will use both fields.